src/box/alter.cc

```diff
@@ -4150,6 +4150,7 @@ on_commit_dd_version(struct trigger *trigger, void * /* event */)
 	struct fiber *fiber = data->fiber;
 	if (fiber != NULL)
 		fiber_wakeup(fiber);
+	box_broadcast_status();
```

Comment:
Is it issued on downstreams too or only on an instance that performs the schema upgrade?

I guess that the former is true. If it is not complicated, I would add a test case that makes it clear.

Reply:
It's executed everywhere, on every instance in the replicaset. Added test for that